octokit-project/src/main/java/org/example/calculator/Calculator.java

```diff
@@ -0,0 +1,15 @@
+package org.example.calculator;
+
+public class Calculator {
+
+    public static void main(String[] args) {
+        int a = 1;
+        int b = 2;
+
+        int c = a + b;
+
```

Comment:
[Severity: Minor] The output message is clear, but it could be improved by including the actual values of 'a' and 'b'.
You can modify the print statement to: 'System.out.println("Sum of " + a + " and " + b + " is: " + c);'. This will provide more context in the output.
Rationale: Providing context in output messages helps users understand the results better.
Reference: https://www.oracle.com/java/technologies/javase/javadoc/index.html